server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -308,6 +308,12 @@ public class ShuffleServerConf extends RssBaseConf {
       .defaultValue(60 * 1000L)
       .withDescription("The timeout of the cache which record the mapping information");
 
+  public static final ConfigOption<Long> SERVER_LEAK_SHUFFLE_DATA_CHECK_INTERVAL = ConfigOptions
```

Comment:
Could you add this config option in our document https://github.com/apache/incubator-uniffle/blob/master/docs/server_guide.md ？